src/interfaces/IEthereumVaultConnector.sol

```diff
@@ -162,7 +164,8 @@ interface IEVC {
     function setPermitDisabledMode(bytes19 addressPrefix, bool enabled) external payable;
 
     /// @notice Sets the nonce for a given address prefix and nonce namespace.
-    /// @dev This function can only be called by the owner of the address prefix. Each nonce namespace provides a 256 bit
+    /// @dev This function can only be called by the owner of the address prefix. Each nonce namespace provides a 256
+    /// bit
```

Comment:
Awkward `forge fmt` artifact? There's a bunch of cases in this file where comments are wrapped like this.

Reply:
unfortunately, that's the case

Reply:
Got it. But you could move stuff from the previous line until it all compacts. I guess I'm fine either way, but don't you think this looks silly?

```
    /// @dev This function can only be called by the owner of the address prefix. Each nonce namespace provides a 256
    /// bit
    /// nonce that has to be used sequentially. There's no requirement to use all the nonces for a given nonce namespace
```

Reply:
of course it looks silly but actually, it's already fixed on `master-working` branch. the reason it looks ugly here is that I forgot to call `forge fmt` on master when freezing the audit commit :) but I'll fix it here too!

Reply:
it was fixed two weeks ago on `master-working`: https://github.com/euler-xyz/ethereum-vault-connector/commit/0f7ad5fe299dde38bdd23dfa5f8998bedb1c5bcf

Reply:
Looks good now thx!